### PR TITLE
fix: use optional chaining for `process.env`

### PR DESCRIPTION
**Pull request by jsonnull**
## 🎯 Changes

When using trpc v10 outside of a server environment, `process` may be undefined. We can pass `isServer: false` in the RuntimeConfig, but the default `isServer` value is created in the module scope regardless, so the `process.env` check will throw.

This simply changes `process.env.VAR` to `globalThis.process?.env?.VAR` where applicable to support non-server environments.

## ✅ Checklist

- [x] I have followed the steps listed in the [Contributing guide](https://github.com/trpc/trpc/blob/main/CONTRIBUTING.md).
- [ ] If necessary, I have added documentation related to the changes made.
- [ ] I have added or updated the tests related to the changes made.

### CODE REVIEW

**Comment by sachinraja** [comment]
Should we just check if `process` exists instead?

**Comment by jsonnull** [comment]
>

**Comment by jsonnull** [comment]
> Should we just check if `process` exists instead?

@sachinraja I can do that, want me to just optionally chain the `process.env.` parts?

**Comment by jsonnull** [comment]
@sachinraja Updated ❤️

**Comment by sachinraja** [comment]
We can't do that either because `process` won't even be a variable. So we'll get some error like `unresolved variable "process"`. I think we need to optionally chain it off of `globalThis`: `globalThis.process?.env`

**Comment by jsonnull** [comment]
Of course! Updated again 😄

**Review by sachinraja — APPROVED** [review]

**Comment by sachinraja** [comment]
Thanks for bearing with me 😄

**Comment by jsonnull** [comment]
Thanks for thinking this through where I just didn't 😂

File: examples/next-prisma-starter-websockets/src/server/context.ts [hunk 1]
```diff
@@ -1,4 +1,3 @@
-import { PrismaClient } from '@prisma/client';
```
Comment (jsonnull) [inline examples/next-prisma-starter-websockets/src/server/context.ts:1]: Drive-by lint fix

**Review by sachinraja — APPROVED** [review]
